authorize/authorize.go

```diff
@@ -154,5 +166,6 @@ func (a *Authorize) OnConfigChange(ctx context.Context, cfg *config.Config) {
 		log.Ctx(ctx).Error().Err(err).Msg("authorize: error updating state")
 	} else {
 		a.state.Store(state)
+		a.groupsCacheWarmer.UpdateConn(state.dataBrokerClientConnection)
```

Comment:
it seems like this would be called on _any_ configuration change - i.e. route/etc, so in a high churn environment where a lot of routes get updated all the time, that will be constantly invalidating the cache?

Reply:
I attempted to account for this. What are you suggesting I do instead?

Reply:
I will update this to only call this function if the connection changes and remove the code I wrote to account for this in the cache warmer.